discord/app_commands/transformers.py

```diff
@@ -387,6 +388,46 @@ def min_value(self) -> Optional[Union[int, float]]:
     @property
     def max_value(self) -> Optional[Union[int, float]]:
         return self._max
+    
+    async def convert(self, ctx, argument):
```

Comment:
What is the return type and the type of both `ctx` and `argument`?

Reply:
ctx is of discord.ext.commands.Context, but I can’t import that because it causes a circular import error. The return type is also dependent on self._opt_type but I suppose a typing.Union[int,float,str] would work

Reply:
Oh and argument should just be a str